### PR TITLE
Add src/30_Advanced/Joining_Analytics_Sessions.html

### PR DESCRIPTION
**Pull request by lannka**
/cc @jeffjose

### CODE REVIEW

File: src/30_Advanced/Joining_Analytics_Sessions.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<!---{
+ "draft": "true"
+}--->
+<!--
+  ## Introduction
+  Linker is an amp-analytics feature to join user sessions across AMP Cache domain and publisher
```
Comment (sebastianbenz) [inline src/30_Advanced/Joining_Analytics_Sessions.html]: Can you add a link to the official docs on ampproject.org.

File: src/30_Advanced/Joining_Analytics_Sessions.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <title>Joining analytics sessions</title>
+  <!-- ## Setup -->
+  <!--  Import the amp-analytics component in the header. -->
+  <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <meta name="amp-google-client-id-api" content="googleanalytics">
```
Comment (sebastianbenz) [inline src/30_Advanced/Joining_Analytics_Sessions.html]: Is this relevant for the sample? If yes, add a comment to make it appear in the doc view.

File: src/30_Advanced/Joining_Analytics_Sessions.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+              "_ga": "CLIENT_ID(_ga)",
+              "gclid": "QUERY_PARAM(gclid)"
+            }
+          }
+        }
+      }
+    </script>
+  </amp-analytics>
+
+  <!-- ## Example Links -->
```
Comment (sebastianbenz) [inline src/30_Advanced/Joining_Analytics_Sessions.html]: Can you explain why these are listed here?

File: src/30_Advanced/Joining_Analytics_Sessions.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  <title>Joining analytics sessions</title>
+  <!-- ## Setup -->
+  <!--  Import the amp-analytics component in the header. -->
+  <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <meta name="amp-google-client-id-api" content="googleanalytics">
+</head>
+<body>
+
+  <!-- ## Basic Usage -->
```
Comment (sebastianbenz) [inline src/30_Advanced/Joining_Analytics_Sessions.html]: Maybe rename heading to `Custom Linker`.

File: src/30_Advanced/Joining_Analytics_Sessions.html [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+            },
+            "destinationDomains": ["www.google.com"],
+            "enabled": true
+          }
+        }
+      }
+    </script>
+  </amp-analytics>
+
+  <!--
```
Comment (sebastianbenz) [inline src/30_Advanced/Joining_Analytics_Sessions.html]: Suggestion, add a second heading: `## Google Analytics`

File: src/30_Advanced/Joining_Analytics_Sessions.html [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+  Google Analytics linker for CID API users
```
Comment (sebastianbenz) [inline src/30_Advanced/Joining_Analytics_Sessions.html]: Can you add a little bit more explanation and make it a full sentence.

File: src/30_Advanced/Joining_Analytics_Sessions.html [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+  <!--  Import the amp-analytics component in the header. -->
+  <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <meta name="amp-google-client-id-api" content="googleanalytics">
+</head>
+<body>
+
+  <!-- ## Basic Usage -->
+  <!--
+  Custom linker
```
Comment (sebastianbenz) [inline src/30_Advanced/Joining_Analytics_Sessions.html]: Can you add a little bit more explanation and make it a full sentence.

**Comment by lannka** [comment]
@sebastianbenz thanks for the comments. yep this is an early draft.  At this point, this is mainly for our own end-to-end testing purpose as the feature isn't fully launched yet, and we're also lacking of official documentation.

I'm going to merge it for now and improve later.

/cc @calebcordry

**Pull request merged**